integrations/DCMFloodlight/browser.js

```diff
@@ -274,6 +202,40 @@ class DCMFloodlight {
     window.gtag("event", "conversion", eventSnippetPayload);
   }
 
+  trackWithIframe(message, salesTag, customFloodlightVariable, countingMethod) {
```

Comment:
please link the public doc

Reply:
already added in `buildIframeTrackPayload`